nautobot/cloud/tests/test_views.py

```diff
@@ -36,6 +37,22 @@ def setUpTestData(cls):
             "comments": "New comments",
         }
 
+    def test_post_without_secrets_group(self):
+        """Assert Secrets Group form field is not required: Fix for https://github.com/nautobot/nautobot/issues/6096"""
```

Comment:
NB: This test is only for testing without secrets group; the test for with a secret group has already been tested in the `setUpTestData` method.